joomla/cassiopeia
# [4.0] Allow for custom prefixes.

Pull Request for Issue # .

### Summary of Changes
Better b/c handling.
Allows for setting custom prefix & icons.

### Testing Instructions
compare & inspect icons.
Should be no changes.
apply pr.
install akeeba backup.
akeeba icon should display in dashboard.
no icon changes should be visible.

### Expected result

![image](https://user-images.githubusercontent.com/1850089/93159344-949beb00-f6d3-11ea-8777-3ade433f2d9b.png)


### Actual result
![image](https://user-images.githubusercontent.com/1850089/93159527-f9574580-f6d3-11ea-8f69-7b29b816c8ce.png)


### Documentation Changes Required

Explain the ability of using prefix.  I don't think this is documented currently.
